examples/README.md

```diff
@@ -1,5 +1,52 @@
+## Example Startup Configurations
+Here are a list of example Sync Gateway Startup configurations found in `examples/startup_configs`:
```

Comment:
It's a bit of a nit, but I think 'examples/startup_config' is better than 'examples/startup_configs' (same for database_configs), and for consistency I think we should use 'legacy_config'.